src/main/java/uk/gov/hmcts/probate/controller/LifeEventController.java

```diff
@@ -37,6 +38,23 @@
         return ResponseEntity.ok(callbackResponseTransformer.updateTaskList(request));
     }
 
+    @PostMapping(path = "/manualUpdateAboutToStart")
+    public ResponseEntity<CallbackResponse> manualUpdateAboutToStart(@RequestBody CallbackRequest request) {
+        final CaseDetails caseDetails = request.getCaseDetails();
```

Comment:
can the details of the update be moved into its own service? ie the transformations involved?